src/pycall.jl

```diff
@@ -43,3 +48,35 @@ end
 function DLArray{T, N}(::Type{A}, ::Type{M}, o::PyObject, to_dlpack) where {T, N, A, M}
     return DLArray{T, N}(A, M, DLManagedTensor(to_dlpack(o)), o)
 end
+
+function wrap(o::PyObject, to_dlpack::Union{PyObject, Function})
+    return wrap(DLManagedTensor(to_dlpack(o)), o)
+end
+#
+function wrap(::Type{A}, ::Type{M}, o::PyObject, to_dlpack) where {
+    T, N, A <: AbstractArray{T, N}, M
+}
+    return wrap(A, M, DLManagedTensor(to_dlpack(o)), o)
+end
+
+share(A::StridedArray, from_dlpack::PyObject) = share(A, PyObject, from_dlpack)
```

Comment:
```suggestion
share(A::StridedArray, from_dlpack::Union{PyObject,Function}) = share(A, PyObject, from_dlpack)
```

Reply:
With this fix, https://github.com/rejuvyesh/PyCallChainRules.jl/pull/10 works quite well!

Reply:
If your'e using `PyCall` and a `from_dlpack::Function` you're supposed to be doing `share(A, PyObject, from_dlpack)`. The issue with this change is that if I want to support both PyCall and PythonCall, and someone loads them at the same time, there's going to be ambiguities.